tests/mail/tests.py

```diff
@@ -913,6 +913,34 @@ def test_attach_non_utf8_text_as_bytes(self):
         self.assertEqual(content, b"\xff")
         self.assertEqual(mimetype, "application/octet-stream")
 
+    def test_attach_8bit_rfc822_message(self):
+        """
+        Attaching a message containing non-ASCII
+        characters does not raise a UnicodeEncodeError,
+        while preserving the integrity of the attached email
+        as a standalone RFC 822 message (#36119)
+        """
+        # The blank line is required so separate header from the body
+        attachment = dedent(
+            """\
+            Content-Type: text/plain; charset=utf-8
+            Content-Transfer-Encoding: 8bit
```

Comment:
To make this technically correct, we actually do need a _Subject_ (or _Date_ or _From_), despite my earlier attempts to minimize the test:

```suggestion
            Subject: A message using 8bit CTE
            Content-Type: text/plain; charset=utf-8
            Content-Transfer-Encoding: 8bit
```